drivers/include/at24xxx.h

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_at24xxx AT24XXX EEPROM unit
+ * @brief       Device driver interface for the AT24XXX EEPROM units
+ *
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for AT24XXX EEPROM units.
+ *
+ * @author      Fabian Hüßler <fabian.huessler@ovgu.de>
+ *
+ */
+
+#ifndef AT24XXX_H
+#define AT24XXX_H
+
+#include <stdint.h>
+
+#include "periph/gpio.h"
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * AT24C128 16kB memory
+ */
+#define AT24C128_EEPROM_SIZE      (16384U)
+
+/**
+ * AT24C256 32kB memory
+ */
+#define AT24C256_EEPROM_SIZE      (32768U)
+
+/**
+ * return values
+ */
+enum {
+    AT24XXX_OK,
+};
+
+/**
+ * 7 bit I2C device address
+ * 10100 A1 A0
+ */
+enum {
+    AT24XXX_DEV_ADDR_00    = (0x50),   //A1 = 0, A0 = 0
```

Comment:
C++ style comments are against RIOT's coding convention. (Yes, they are part of C since C99 and are shorter to write. I personally prefer them also, but let's stick with the coding convention here :-))

Note:

```C
// This is a C++ style code
/* This is a C style comment */
```

Also, please use Doxygen comments here, e.g.

```C
    AT24XXX_DEV_ADDR_00    = (0x50),   /**< I2C address when pins A1 and A2 pulled low */
```